manifests/database/postgresql.pp

```diff
@@ -41,7 +41,7 @@
 
   if $init_db {
     $ssl_verify_options = $db_ssl_verify ? {
-      false => '&sslfactory=org.postgresql.ssl.NonValidatingFactory',
+      false => '\\&sslfactory=org.postgresql.ssl.NonValidatingFactory',
```

Comment:
I'm a bit hesitant about this one. If we do this, fixing it in cpdb becomes much harder. If we ensure that when adding escaping, we also add SSL parameters. It would clean up this code.